src/low-level/mime/mailmime_content.c

```diff
@@ -1837,6 +1860,9 @@ int mailmime_quoted_printable_body_parse(const char * message, size_t length,
 
 	case '\r':
 	  if (cur_token + 2 >= length) {
+      if (!partial) {
```

Comment:
did you mean `if (partial)` here?
or you did you change a behavior?
or did you fix an issue?